cmd/server-main.go

```diff
@@ -482,6 +483,14 @@ func serverMain(c *cli.Context) {
 		fatalIf(errors.New("empty directory"), "Configuration directory cannot be empty.")
 	}
 
+	// Disallow relative paths, figure out absolute paths.
```

Comment:
Since we use fatalIf currently style is more preferred. It collapses the `if err != nil { `